dev_tools/docker/apline/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine:latest
+
+# Install dependencies.
+RUN apk upgrade --update && \
+    apk add --no-cache bash git emacs vim && \
+    apk add libpng-dev freetype-dev openblas-dev libc-dev musl-dev && \
+    apk add python3-dev python3-tkinter && \
+    python3 -m pip install --upgrade pip && \
+    pip3 install --no-cache --upgrade pip setuptools wheel
+
+# Make python3 default
+RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
+
+# Install Cirq library from Github.
+RUN git clone https://github.com/quantumlib/Cirq
+# Install the needed Python libraries.
+RUN pip3 install -r Cirq/requirements.txt -r Cirq/cirq/contrib/contrib-requirements.txt -r Cirq/dev_tools/conf/pip-list-dev-tools.txt
```

Comment:
`python -m pip` is more reliable than `pip3` in my experience, though I guess in the context of a dockerfile you know it would keep working.

Reply:
I agree with you since other Docker file in our project was following pip3 command installation. I was trying to make a generic as it was maintained in other Docker file.
I would suggest implementing some changes to other docker files as well @Strilanc